mod/filerm.php

```diff
@@ -21,7 +21,7 @@ function filerm_content(&$a) {
 		file_tag_unsave_file(local_user(),$item_id,$term, $category);
 
 	if(x($_SESSION,'return_url'))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊